keyboards/keebio/iris/keymaps/jestes5111/keymap.c

```diff
@@ -0,0 +1,184 @@
+  //├────────┼────────┼────────┼────────┼────────┼────────┤                          ├────────┼────────┼────────┼────────┼────────┼────────┤
+     RGB_TOG, KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_LPRN,                            KC_RPRN, VM,      INTEG,   KC_NO,   KC_NO,   RESET,
+  //├────────┼────────┼────────┼────────┼────────┼────────┤                          ├────────┼────────┼────────┼────────┼────────┼────────┤
+     FILL,    KC_MPRV, KC_MNXT, KC_VOLU, KC_NO,   KC_LBRC,                            KC_RBRC, KC_NO,   KC_NO,   KC_NO,   RGB_VAI, KC_NO,
+  //├────────┼────────┼────────┼────────┼────────┼────────┼────────┐        ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+     KC_MUTE, KC_MSTP, KC_MPLY, KC_VOLD, KC_NO,   KC_LCBR, KC_MENU,          KC_NO,   KC_RCBR, KC_END,  RGB_HUD, RGB_SAD, RGB_VAD, KC_NO,
+  //└────────┴────────┴────────┴───┬────┴───┬────┴───┬────┴───┬────┘        └───┬────┴───┬────┴───┬────┴───┬────┴────────┴────────┴────────┘
+                                    KC_NO,   UNLOCK,  KC_UNDS,                   KC_NO,   KC_NO,   KC_NO
+                                // └────────┴────────┴────────┘                 └────────┴────────┴────────┘
+  ),
+
+  [_FUNC] = LAYOUT(
+  //┌────────┬────────┬────────┬────────┬────────┬────────┐                          ┌────────┬────────┬────────┬────────┬────────┬────────┐
+     KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,                              KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS, KC_NO,   KC_BSPC,
+  //├────────┼────────┼────────┼────────┼────────┼────────┤                          ├────────┼────────┼────────┼────────┼────────┼────────┤
+     QWERTY,  KC_WH_L, KC_WH_D, KC_WH_U, KC_WH_R, KC_BTN3,                            KC_P7,   KC_P8,   KC_P9,   KC_PPLS, KC_NO,   KC_NO,
+  //├────────┼────────┼────────┼────────┼────────┼────────┤                          ├────────┼────────┼────────┼────────┼────────┼────────┤
+     KC_BTN4, KC_MS_L, KC_MS_D, KC_MS_U, KC_MS_R, KC_NO,                              KC_P4,   KC_P5,   KC_P6,   KC_NO,   KC_NO,   KC_NO,
+  //├────────┼────────┼────────┼────────┼────────┼────────┼────────┐        ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+     KC_BTN5, KC_NO,   KC_NO,   KC_NO,   KC_BTN2, KC_NO,   KC_LALT,          KC_NO,   KC_P1,   KC_P2,   KC_P3,   KC_NO,   KC_NO,   KC_NO,
+  //└────────┴────────┴────────┴───┬────┴───┬────┴───┬────┴───┬────┘        └───┬────┴───┬────┴───┬────┴───┬────┴────────┴────────┴────────┘
+                                    KC_NO,   KC_NO,   KC_BTN1,                   KC_PENT, KC_P0,   KC_PDOT
+                                // └────────┴────────┴────────┘                 └────────┴────────┴────────┘
+  )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  if (record->event.pressed) {
+    switch(keycode) {
+    // default layer
+    case QWERTY:
+        layer_off(_FUNC);
+        default_layer_set(_QWERTY);
+        return false;
+
```

Comment:
You are not using any other layers as the default layer.  This functionaly can/should be removed.